spec/system/candidate_interface/helpers/candidate_helper.rb

```diff
@@ -0,0 +1,37 @@
+module CandidateHelper
+  def given_i_am_signed_in
```

Comment:
I've found that re-using steps between feature tests make the tests harder to read.

If duplication is a problem (might not be!), I'd prefer to keep the step definitions in the feature specs, but make them use a helper.

For example:

```rb
# candidate_viewing_application_spec.rb 
def given_i_am_signed_in
  sign_in_candidate
end
```

And the helper

```rb
# candidate_helper.rb
module CandidateHelper
  def sign_in_candidate
    candidate = create(:candidate)
    login_as(candidate)
  end
end
```

Reply:
I got the point. What I am gonna do is keep the duplication for the small methods,
and "dry" only the big ones (eg `and_i_filled_in_personal_details` is one of this)